infrastructure/template.yaml

```diff
@@ -555,6 +555,165 @@ Resources:
         - !Ref AlarmNotificationTopic
       ActionsEnabled: true
 
+  ######################
+  # Deleting the Record
+  ######################
+
+  UserAccountDeletionTopic:
+    Type: AWS::SNS::Topic
+    Properties:
+      DisplayName: UserAccountDeletionTopic
+      KmsMasterKeyId: !Ref SnsKmsKey
+      TopicName: UserAccountDeletion
+
+  DeteUserServicesSubscription:
+    Type: AWS::SNS::Subscription
+    Properties:
+      Endpoint: !GetAtt DeleteUserServicesFunction.Arn
+      Protocol: lambda
+      RedrivePolicy:
+        deadLetterTargetArn: !GetAtt UserAccountDeletionTopicDeadLetterQueue.Arn
+      TopicArn: !Ref UserAccountDeletionTopic
+
+  DeleteUserServicesFunction:
+    DependsOn:
+      - DeleteUserServicesFunctionLogGroup
+    Type: AWS::Serverless::Function
+    Properties:
+      Architectures: ["x86_64"]
```

Comment:
Could we include a Function Name here for the log groups please  - https://github.com/alphagov/di-account-management-backend/blob/main/infrastructure/template.yaml#LL119C7-L119C19

Reply:
ah excellent! thanks was wondering why yours were named differently to mine!

Reply:
Done:
https://github.com/alphagov/di-account-management-backend/pull/36/files#diff-1d6a0a19507d7cdddd8ff209f817d2d18f3fb22ceec6b0eb332b0a14292c69b0R583